experimental/origin-graphql/src/typeDefs/Growth.js

```diff
@@ -27,16 +27,28 @@ module.exports =
   }
 
   enum GrowthActionType {
-    basicProfile
-    attestations
+    email
+    phoneNumber
```

Comment:
supernit. maybe we could just call this one phone.

Reply:
yup better, changing